ipwb/replay.py

```diff
@@ -1067,7 +1067,7 @@ def start(cdxj_file_path, proxy=None, port=IPWBREPLAY_PORT):
         print((f'IPWB replay started on '
                f'http://{host_port[0]}:{host_port[1]}'))
 
-        app.run(host='0.0.0.0', port=host_port[1])
+        app.run(host=host_port[0], port=host_port[1])
```

Comment:
I think it is better to leave it `0.0.0.0`, if not then we should add a special case where we define things in utils.

Reply:
It really depends on the purpose of the `host` parameter. If it is to be used for formatting FQDN URIs then it really does not matter which interface the service is listening to, as long as it is reachable, which means running it on `0.0.0.0` will do, irrespective of what hostname is used.

Reply:
Ok, reverted in [81ae262](https://github.com/oduwsdl/ipwb/pull/766/commits/81ae2622a261bca6b3843e48bb8ff99e85250a41), but the specified host still does not appear in the TimeMaps. I will move this PR to WIP for now until that issue is resolved for this branch.